concepts/conditionals-switch/introduction.md

```diff
@@ -0,0 +1,30 @@
+# Introduction
+
+Besides the if-statement, JavaScript also has a switch-statement to conditionally execute logic.
+It is used when a single variable needs to be compared to multiple variants.
+The comparison is done by checking for strict equality (`===`), see [concept comparison][concept-comparison].
+For some variable `x`, the switch statement in JavaScript has the following syntax.
+
+<!-- prettier-ignore-start -->
+```javascript
+switch (x) {
+  case option1:
+    // code that is executed when "x === option1" is true
+    break;
+  case option2:
+    // code that is executed when "x === option2" is true
+    break;
+  default:
+    // code that is executed when x does not equal any of the options
+}
+```
+<!-- prettier-ignore-end -->
+
+The `default` case is optional and used in case you want to execute some code if none of the other options match the variable.
```

Comment:
```suggestion
The `default` case is optional and used when you want to execute some code if none of the other options match the variable.
```

(case in case haha)